tendrl/gluster_integration/flows/enable_disable_volume_profiling/__init__.py

```diff
@@ -0,0 +1,107 @@
+from tendrl.commons import flows
+from tendrl.commons.flows.exceptions import FlowExecutionFailedError
+from tendrl.commons.utils import cmd_utils
+from tendrl.commons.utils import log_utils as logger
+
+
+VOL_PROFILE_ACTIONS = {
+    "enable": "start",
+    "disable": "stop"
+}
+
+
+class EnableDisableVolumeProfiling(flows.BaseFlow):
+    def __init__(self, *args, **kwargs):
+        super(EnableDisableVolumeProfiling, self).__init__(
+            *args,
+            **kwargs
+        )
+
+    def run(self):
+        action = self.parameters["Cluster.volume_profiling_flag"]
+        if action not in VOL_PROFILE_ACTIONS.keys():
+            raise FlowExecutionFailedError(
+                "Invalid value of Cluster.volume_profiling_flag "
```

Comment:
Please mention in log what values are valid i.e "enable" | "disable"

Reply:
Done